v5_account_service.go

```diff
@@ -86,3 +101,21 @@ func (s *V5AccountService) GetWalletBalance(at AccountType, coins []Coin) (*V5Wa
 
 	return &res, nil
 }
+
+// GetAccountInfo :
+func (s *V5AccountService) GetAccountInfo(unifiedMarginStatus int, marginMode MarginMode, updatedTime string) (*V5AccountInfoResponse, error) {
```

Comment:
Request Parameters does not seem necessary.

<img width="768" alt="スクリーンショット 2023-04-10 21 58 44" src="https://user-images.githubusercontent.com/12891407/230906060-cdb2e147-010f-4224-b7b9-37350891f6c0.png">


```suggestion
func (s *V5AccountService) GetAccountInfo() (*V5AccountInfoResponse, error) {
```

Reply:
I will fix it after the merge.

Reply:
Ah that's my bad. Thanks!